Realm/RLMSyncSubscription.h

```diff
@@ -117,13 +117,11 @@ RLM_HEADER_AUDIT_BEGIN(nullability, sendability)
 - (void)write:(__attribute__((noescape)) void(^)(void))block __attribute__((unavailable("Renamed to -update")));
 
 /**
- Synchronously performs any transactions (add/remove/update) to the subscription set within the block,
- this will not wait for the server to acknowledge and see all the data associated with this collection of subscriptions,
- and will return after committing the subscription transactions.
+ Synchronously performs any transactions (add/remove/update) to the subscription set within the block. The `onComplete` block is executed after waiting for associated data to be downloaded from the server.
 
  @param block The block containing actions to perform to the subscription set.
  @param onComplete A block which is called upon synchronization of
-                   subscriptions to the server. The block will be passed `nil`
+                   subscriptions from the server. The block will be passed `nil`
```

Comment:
to is correct here?

Reply:
I was thinking "the data _from_" when I changed it.

Reply:
Changing it to "...A block which is called upon synchronization of
                   data from the server..."